core/base/document.php

```diff
@@ -1258,6 +1259,120 @@ public function __construct( array $data = [] ) {
 		parent::__construct( $data );
 	}
 
+	/*
+	 * Get Export Data
+	 *
+	 * Filters a document's data on export
+	 *
+	 * @since 3.2.0
+	 * @access public
+	 *
+	 * @return array The data to export
+	 */
+	public function get_export_data() {
+		$content = Plugin::$instance->db->iterate_data( $this->get_elements_data(), function( $element_data ) {
+			$element_data['id'] = Utils::generate_random_string();
```

Comment:
It will destroy custom CSS with the element id selector., why it's needed?

Reply:
This is the same behavior as it was before in templates export